docs/install.md

```diff
@@ -164,48 +164,48 @@ Information](#set-system-revision-information)).
 
 #### Procedure
 
-1. Ensure the files are readable only by `root`.
+1. (`ncn-m001#`) Ensure the files are readable only by `root`.
 
     ```screen
-    ncn-m001# touch /root/.config/sat/s3_access_key \
+    touch /root/.config/sat/s3_access_key \
         /root/.config/sat/s3_secret_key
     ```
 
     ```screen
-    ncn-m001# chmod 600 /root/.config/sat/s3_access_key \
+    chmod 600 /root/.config/sat/s3_access_key \
         /root/.config/sat/s3_secret_key
     ```
 
-1. Write the credentials to local files using `kubectl`.
+1. (`ncn-m001#`) Write the credentials to local files using `kubectl`.
 
    ```screen
-   ncn-m001# kubectl get secret sat-s3-credentials -o json -o \
+   kubectl get secret sat-s3-credentials -o json -o \
        jsonpath='{.data.access_key}' | base64 -d > \
        /root/.config/sat/s3_access_key
    ```
 
    ```screen
-   ncn-m001# kubectl get secret sat-s3-credentials -o json -o \
+   kubectl get secret sat-s3-credentials -o json -o \
        jsonpath='{.data.secret_key}' | base64 -d > \
        /root/.config/sat/s3_secret_key
    ```
 
 1. Verify the S3 endpoint specified in the SAT configuration file is correct.
 
-   1. Get the SAT configuration file's endpoint value.
+   1. (`ncn-m001#`) Get the SAT configuration file's endpoint value.
 
       **Note:** If the command's output is commented out, indicated by an initial `#`
       character, the SAT configuration will take the default value – `"https://rgw-vip.nmn"`.
 
       ```screen
-      ncn-m001# grep endpoint ~/.config/sat/sat.toml
+      grep endpoint ~/.config/sat/sat.toml
       # endpoint = "https://rgw-vip.nmn"
```

Comment:
This line is output. I'll stop commenting on each of these now. I think you'll be able to tell which lines are commands from here. Let me know if there are any questions.

Basically, if a line in a code block does not begin with a command prompt like `ncn-m001# ` and it is not a continuation of a command started on previous line (i.e. the previous line ends with a backslash), then it is output from the command.

Reply:
Updated in next commit.